docs/pages/account/setup-email.mdx

```diff
@@ -0,0 +1,33 @@
+import { Callout } from "nextra/components";
+import { Steps } from "nextra/components";
+
+# Mein Konto mit einer E-Mail verknüpfen
+
+<Callout type="warning">
+  Die Anmeldung ist aus datenschutzrechtlichen Gründen erst ab 16 Jahren erlaubt!
+</Callout>
+
+Damit deine Daten immer und überall erreichbar sind, musst du deinen Account mit einer E-Mail verknüpfen.  
+Du provitierst dann von folgenden Dingen:
+
+- Automatisches Backup
+  Weiterhin Zugriff auf die Daten bei Verlusst deines Gerätes
```

Comment:
<img width="655" alt="image" src="https://github.com/user-attachments/assets/e397b316-c9d5-4c25-8fca-f465526b1043">


You need to add I think a `\` I think for a line break. Or we just change it one line.

```suggestion
- Automatisches Backup: Weiterhin Zugriff auf die Daten bei Verlusst deines Gerätes
```